hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/Gateway.java

```diff
@@ -63,13 +69,35 @@ public static void main(String[] args) throws Exception {
     new Gateway().run(args);
   }
 
+  private void setHttpBaseDir(OzoneConfiguration ozoneConfiguration)
```

Comment:
```
hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/Gateway.java
 72: 'ozoneConfiguration' hides a field.
```

I guess the parameter is unnecessary, it can use the field directly.

Reply:
done